RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -3538,10 +3538,17 @@ float [stat] provideStats(int [stat] amt, boolean doEquips, boolean speculative)
 		Blessing of Your Favorite Bird,
 	]))
 		return result();
+
 	if(auto_have_skill($skill[Quiet Desperation]))
 		tryEffects($effects[Quiet Desperation]);
 	else
 		tryEffects($effects[Disco Smirk]);
+
+	if (!pass() && (0 == have_effect($effect[Triple-Sized])))
+	{
+		auto_powerfulGloveStats();
+	}
+
```

Comment:
I'd much rather add Triple-Sized to buffMaintain so that speculative mode works with this as well. Let me know if you need some advice on how to do that.